README.md

```
@@ -88,7 +90,9 @@ PROCESSING_DATE_TIME|MESSAGE_ID|TRANSACTION_TYPE|PAYMENT_CURRENCY_CODE|TOTAL_PAY
 | TRANSACTION_TYPE |	PmtTpInf.CtgyPurp.Prtry | Underlying reason for the payment transaction.
 | PAYMENT_CURRENCY_CODE |	InstdAmt.Amt.Ccy | The currency of the instructed amount as ordered by the initiating party.
 | TOTAL_PAYMENT_AMOUNT |	InstdAmt.Amt.Amt | Amount of money to be moved between the debtor and creditor, before deduction of charges, expressed in the currency as ordered by the initiating party.
+| SENDER_ID |	Dbtr.Id | Unique identifier by which the debtor is known and which is usually used to identify that party.
```

markdownlint-cli2 (0.22.1)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)